monero-wallet-gui.pro

```diff
@@ -144,26 +144,36 @@ CONFIG(WITH_SCANNER) {
 
 win32 {
 
+    # QMAKE_HOST.arch is unreliable, will allways report 32bit if mingw32 shell is run.
+    # Obtaining arch through uname should be reliable. This also fixes building the project in Qt creator without changes.
+    MSYS_HOST_ARCH = $$system(uname -a | grep -o "x86_64")
+
+    # Even if host is 64bit, we are building win32 spec, so mingw path should allways be "/mingw32"
```

Comment:
@rpcjacobs What does this mean, building "win32 spec"? Previously my static boost libs were in /mingw64/boost/lib and aren't found with this change. Its no big deal, I'll adjust but I'm curious.